### DISCUSSION TITLE
tx2i doflash error after upgrade to Honister (master)

### DISCUSSION THREAD

**Discussion by ddbabich** [OE4T/meta-tegra#809] [Q&A] [answered]
Hi, I recently upgraded from the dunfell tag to honister (the current meta-tegra 'master').  When I built my custom tx2i project I received an error during doflash.  So I decided to try and just build the tegra-demo-disto demo-image-full image to see if the problem was with my custom project or with the latest release.  When I attempt to flash the demo-full project to the devkit, I also received the same error.  I'm not sure what's changed that is causing it.  Any help is greatly appreciated.  Here is the error:
```
Welcome to Tegra Flash
version 1.0.0
Type ? or help for help and q or quit to exit
Use ! to execute system commands
 
[   0.0018 ] Generating RCM messages
[   0.0093 ] tegrarcm_v2 --listrcm rcm_list.xml --chip 0x18 0 --download rcm mb1_recovery_prod.bin 0 0
[   0.0114 ] RCM 0 is saved as rcm_0.rcm
[   0.0119 ] RCM 1 is saved as rcm_1.rcm
[   0.0119 ] List of rcm files are saved in rcm_list.xml
[   0.0119 ] 
[   0.0119 ] Signing RCM messages
[   0.0154 ] tegrasign_v3.py --key None --list rcm_list.xml --pubkeyhash pub_key.key
[   0.0155 ] Assuming zero filled SBK key
[   0.0448 ] Copying signature to RCM mesages
[   0.0476 ] tegrarcm_v2 --chip 0x18 0 --updatesig rcm_list_signed.xml
[   0.0497 ] 
[   0.0497 ] Boot Rom communication
[   0.0527 ] tegrarcm_v2 --chip 0x18 0 --rcm rcm_list_signed.xml
[   0.0544 ] BR_CID: 0x81801001643d78420000000017038200
[   0.0554 ] RCM version 0X180001
[   0.5264 ] 
Error: Return value -9
Command tegrarcm_v2 --chip 0x18 0 --rcm rcm_list_signed.xml
ERR: could not retrieve EEPROM board information
```
Thanks
-David

**Comment by madisongh** [comment]
Which dunfell branch were you using (i.e., which L4T version)?   I don't have a TX2i to test with, so it's possible that there's something in the latest R32.6 support that needs fixing to work with that particular module.  I've never seen a `-9` error coming from the flashing tools, so I'm not sure what the problem might be. You may need to manually walk through the same commands that the script is using, comparing the working one against the non-working one, to see what might be different between the two.

**Comment by ddbabich** [comment]
Hi, Sorry I need to correct my original comment.  I was actually last on the zeus-l4t-r32.3.1 tag, though we had previously been on a dunfell tag but had to go back due to a compatibility issue with some dependencies.  That being said, both the dunfell and zeus tags doflash scripts worked.  I will try the manual walk through that you've suggested to see if anything stands out and let you know what I find.  Thanks.

**Comment by ddbabich** [comment, marked as answer]
Oh boy.  This turned out to be a network AI security scanner killing my flash attempts.  I think the new security features added to the jetpack-4.6's flashing procedure were raising an alarm.  So whenever I tried to flash it, the security bot would kill my process.  I sure went down a rabbit hole with that one.  But glad it was me and not you.  Thanks @madisongh for the quick response to this though.

**Reply by madisongh** [reply]
Yikes! Well, I'm glad you figured it out.  Thanks for the follow-up!
